### PR TITLE
14806 Add new account type and auth for SBC Staff

### PR DESCRIPTION
**Pull request by ChasBoucher**
*Issue #:* /bcgov/entity#14806

*Description of changes:*
- Add SBC_STAFF to AccountTypes
- Change method of authenticating SBC_STAFF to use AccountType
- Change timeout on RegistrationTable.spec.ts to 3000. Was timing out.
- Minor linting

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/utils/auth-helper.ts [hunk 1]
```diff
@@ -181,8 +181,7 @@ export async function getSbcFromAuth (): Promise<boolean> {
         if (!data) {
           return false
         }
-        const branchName = data?.branchName
-        if (branchName?.includes('Service BC')) {
+        if (data.orgType === AccountTypes.SBC_STAFF) {
```
Comment (cameron-eyds) [inline ppr-ui/src/utils/auth-helper.ts:184]: Just to verify, this `orgType` has been set in all deployed environments? 
ie when this change goes to PROD, SBC Staff will still have their access?

**Review by cameron-eyds — APPROVED** [review]
Code looks fine to me! 
Just had the one question

File: ppr-ui/src/utils/auth-helper.ts (hunk 1 above)
Comment (ChasBoucher) [reply ppr-ui/src/utils/auth-helper.ts:184]: I am assuming so... The change has already been made to the API and I believe it is in production..

File: ppr-ui/tests/unit/RegistrationTable.spec.ts [hunk 2]
```diff
@@ -278,9 +278,9 @@ describe('Test registration table with results', () => {
           startDate: null,
           status: ''
         })
-      }, 1000)
+      }, 3000)
```
Comment (dimak1) [inline ppr-ui/tests/unit/RegistrationTable.spec.ts:281]: I see the comment it was timing out... What about 1.5s or 2s?

**Review by dimak1 — APPROVED** [review]

File: ppr-ui/tests/unit/RegistrationTable.spec.ts (hunk 2 above)
Comment (ChasBoucher) [reply ppr-ui/tests/unit/RegistrationTable.spec.ts:281]: I don't mind experimenting with different times, but I think 3 is a happy medium.  Different machines will give different results.  The main area the test is failing is locally... For example, when I changed to 1 second the test failed..  It sometimes fails with 3 seconds as well.

Comment (dimak1) [reply ppr-ui/tests/unit/RegistrationTable.spec.ts:281]: hmm i see... testing UI with a 3s timeout goes beyond what unit test suppose to do, especially if it still fails sometimes (as you say). But I guess, let's monitor and adjust in the future if need be.

**Review by dimak1 — APPROVED** [review]

**Pull request merged**